android/gradle.properties

```diff
@@ -11,5 +11,5 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 #Fri Apr 23 09:56:57 AEST 2021
-android.enableJetifier=true
+#android.enableJetifier=true
```

Comment:
If it is no longer required, I recommend to remove it instead of commenting it out.